### PR TITLE
dependency update to support flutter 3.22.0 +

### PR DESCRIPTION
**Pull request by nicks258**
## Connection with issue(s)

Close #[62](https://github.com/flutter-form-builder-ecosystem/form_builder_phone_field/issues/62)

Connected Flutter 3.22.0 +

## Solution description
Updated child dependency

- phone_number: ^2.1.0
- flutter_lints: ^4.0.0
- dart sdk: ">=3.3.0 <4.0.0"

### CODE REVIEW

File: example/pubspec.yaml [hunk 1]
```diff
@@ -4,8 +4,7 @@ publish_to: "none" # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=3.0.0 <4.0.0"
-  flutter: ">=3.10.0"
```
Comment (deandreamatias) [inline example/pubspec.yaml:8]: Please add this line 

```dart
flutter: ">=3.22.0"
```

**Review by deandreamatias — CHANGES_REQUESTED** [review]

File: example/pubspec.yaml (hunk 1 above)
Comment (nicks258) [reply example/pubspec.yaml:8]: Done with adding flutter constrain.

**Review by deandreamatias — APPROVED** [review]

**Comment by deandreamatias** [comment]
Now will be required that pass pipeline and I will merge it

**Comment by dthuesen** [comment]
Since its only a matter of the small analyzer issue in the pipeline "Convert 'key' to a super parameter • example/lib/main.dart:12:9 • use_super_parameters", how long do you think it will take to fix it?

**Comment by nicks258** [comment]
Hi @deandreamatias I have push the commit to solve analyzer issue. Can you merge it now. Thanks.

**Comment by nicks258** [comment]
@deandreamatias updated PhoneNumberKit tag to 3.6.0 to solve pod issue. I Hope this will be merge now (;

**Pull request merged**